src/bp/ui-studio/src/web/views/FlowBuilder/skills/index.jsx

```diff
@@ -140,8 +140,16 @@ class SkillsBuilder extends React.Component {
       .then(({ data }) => data)
   }
 
+  findInstalledSkill() {
+    const skillId = this.props.skillId
+    if (skillId) {
+      return find(this.props.installedSkills, x => x.id.toLowerCase() === skillId.toLowerCase())
+    }
+    return undefined
```

Comment:
For me "guard statement" like `if (!skillId) { return }` would be more obvious.

Reply:
@epaminond right ! I went ahead and made the changes